### PR TITLE
Rename playbook for ad-hoc reporting box management

### PR DESCRIPTION
**Pull request by maxrothman**
Though the name "tools-gp" is now a misnomer, a new on-call devops would
probably know the place where read-replica scripts live by that name.
Therefore, until the server is renamed, the playbook should match it.

@e0d

### CODE REVIEW

**Comment by e0d** [comment]
@edx/devops @arbabnazar please note this change.

**Comment by e0d** [comment]
:+1:

**Comment by jibsheet** [comment]
Please update https://openedx.atlassian.net/wiki/display/EdxOps/Add+a+user+to+the+VPN+and+Read+Replica after it's renamed.

**Comment by maxrothman** [comment]
@jibsheet That documentation would've been useful 15 minutes ago (doh)
Updated.

File: playbooks/roles/ad_hoc_reporting/defaults/main.yml [hunk 1]
```diff
@@ -18,8 +18,6 @@
 # users created on the box which is
 # COMMON_USER_INFO + AD_HOC_REPORTING_USER_INFO
 
-AD_HOC_REPORTING_USERS: []
```
Comment (feanil) [inline playbooks/roles/ad_hoc_reporting/defaults/main.yml:21]: Is this var not used?

Comment (maxrothman) [inline playbooks/roles/ad_hoc_reporting/defaults/main.yml:21]: Nope. Did a top-level grep in edx-secure and configuration.

**Comment by feanil** [comment]
:+1:

**Pull request merged**